apps/right-to-rent-check/behaviours/tenancy-start-west-midlands-check.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+const westMidPostcodes = require('../../../west-midland-postcodes.js');
+const pilotStartDate = new Date('2014/12/01').getTime();
+const pilotEndDate = new Date('2016/01/31').getTime();
+
+module.exports = superclass => class extends superclass {
+  saveValues(req, res, callback) {
+    let tenancyStart = req.form.values['tenancy-start'];
+    tenancyStart = new Date(tenancyStart).getTime();
+    const currentPostcode = req.sessionModel.get('property-address-postcode');
+    req.sessionModel.set('valid-tenancy', true);
```

Comment:
do you need to set `valid-tenancy`? Could you just set it if the following conditions aren't met?